src/CMakeLists.txt

```
@@ -11,8 +11,8 @@ protobuf_generate_cpp(msgs_SRCS msgs_HDR msgs.proto)
 include_directories(${CMAKE_CURRENT_BINARY_DIR})
 
 rock_library(gazebo_usv
-    SOURCES USVPlugin.cpp Thrusters.cpp Thruster.cpp Actuators.cpp Utilities.cpp Rudder.cpp Wind.cpp Wave.cpp DirectForceApplication.cpp ${msgs_SRCS}
-    HEADERS USVPlugin.hpp Thrusters.hpp Thruster.hpp Actuators.hpp Utilities.hpp Rudder.hpp Wind.hpp Wave.hpp DirectForceApplication.hpp
+    SOURCES USVPlugin.cpp Thrusters.cpp Thruster.cpp Actuators.cpp RockGazeboHelpers.cpp Rudder.cpp Wind.cpp Wave.cpp DirectForceApplication.cpp ${msgs_SRCS}
+    HEADERS USVPlugin.hpp Thrusters.hpp Thruster.hpp Actuators.hpp Rudder.hpp Wind.hpp Wave.hpp DirectForceApplication.hpp
```

```suggestion
    HEADERS USVPlugin.hpp Thrusters.hpp Thruster.hpp Actuators.hpp RockGazeboHelpers.hpp Rudder.hpp Wind.hpp Wave.hpp DirectForceApplication.hpp
```
Or this is not needed? If so, why?

---

It's a private header that should not be used in other headers ... in that case, it does not have to be installed.